packages/react/src/forms/FormControl/FormControl.module.css

```diff
@@ -116,3 +116,8 @@
     transform: translateY(0);
   }
 }
+
+.FormControl-validation-checkbox {
+  position: absolute;
+  top: 45px;
+}
```

Comment:
👋 @MelissaPastore 

> What do you think of this approach of using CSS to fix this issue? 
> I'm not sure if it's good practice to have a position: absolute element within a flex container?

Thanks for calling this out ✨. While it does work with `position: absolute`, you could also use a grid to achieve a similar effect without needing to manually set a the vertical offset, or change the markup.

E.g. something like this...

<table>

<tr>
<td valign="top">

<!-- Insert "before" screenshot here -->
![Screenshot 2024-09-09 at 11 06 37](https://github.com/user-attachments/assets/8993c920-06bb-4d8b-b829-bed2e99ec98e)


 </td>
<td valign="top">

<!-- Insert "after" screenshot here -->
![Screenshot 2024-09-09 at 11 07 36](https://github.com/user-attachments/assets/d4fc76f0-f3fe-48bd-a55a-d12a094eb547)
</td>
</tr>
</table>

You'll notice that you still need to apply some optical alignment to the checkbox, but this should only be around 1-2px. Another advantage to this approach is that it will preserve the `gap` that's already set in `FormControl`

Here's the CSS I used in the inspector to achieve the layout above, but it can likely be optimized further...

```css
.formcontrol-parent {
  display: grid;
  grid-template-columns: auto 1fr;
}

label {
  grid-column: 2; 
  grid-row: 1;
}

input {
  grid-row: 1; 
}

.validation {
  grid-column: 1 / 3;
}
```

Hope this helps!

Reply:
Thanks for the feedback. I'll try this approach!

Reply:
+1 to @rezrah's grid approach 🙂

Looks great though @MelissaPastore! ✨